test/src/test/java/hudson/search/SearchTest.java

```diff
@@ -64,15 +66,22 @@ public class SearchTest {
 
     @Rule public JenkinsRule j = new JenkinsRule();
 
-    /**
-     * No exact match should result in a failure status code.
-     */
+    private void searchWithoutNavigating(HtmlPage page, String query) throws IOException {
```

Comment:
Could be moved to `jenkins-test-harness` but unsure if it has any use outside of this test class.